server/models/ConfigModel/index.js

```diff
@@ -28,11 +28,21 @@ const ConfigModel = {
 
     return value
   },
-  index: async (db, userId) =>
-    await db
-      .collection(collections.configs)
-      .find({ $or: [{ readers: userId }, { public: true }] })
-      .stream(),
+  all: async (db, userId, activeOnly) =>
+    activeOnly
+      ? await db
+          .collection(collections.configs)
+          .aggregate([
+            { $match: { $or: [{ readers: userId }, { public: true }] } },
+            {
+              $match: { $or: [{ status: { $exists: false } }, { status: 1 }] },
```

Comment:
would this logic be easier if we looked for configs where the status is not 0?

Reply:
it becomes the same query, we would be replacing `{ status: 1 }` with `{ status: { $neq: 0 } }`